pom.xml

```
@@ -224,7 +224,7 @@
         <payara-arquillian-container.version>3.0.alpha4</payara-arquillian-container.version>
         <payara.security-connectors.version>3.0.alpha4</payara.security-connectors.version>
         <opentracing.version>0.33.0</opentracing.version>
-        <h2db.version>1.4.200</h2db.version>
+        <h2db.version>2.1.212</h2db.version>
```

If we are using a patched version of h2db, the version should be the following 
```suggestion
        <h2db.version>2.1.212.payara-p1</h2db.version>
```
I am also not seeing any changes in Payara's h2db project.

---

It is using patched binary by downloading `.zip` file directly from H2 Github and then uploading it to Payara Nexus instead of using neither [Payara's h2db project ](https://github.com/payara/h2db) nor [Payara_PatchedProjects](https://github.com/payara/Payara_PatchedProjects) that already deprecated. There is no change to the code, it is just a repackage module in Payara that requires a `.zip` package of the H2 dependency instead of `.jar` package